stripe/src/main/java/com/stripe/android/model/SetupIntent.kt

```diff
@@ -206,8 +206,6 @@ data class SetupIntent internal constructor(
     }
 
     companion object {
-
-        @JvmStatic
```

Comment:
It might be nice for our Java users to be able to access SetupIntent.fromJson instead of SetupIntent.Companion.fromJson().  Let's keep the JvmStatic annotation.

Reply:
I removed because PaymentIntent didn't have that. Adding to both.